packages/hash/api/src/graphql/typeDefs/ontology/entity-type.typedef.ts

```diff
@@ -33,77 +24,27 @@ export const entityTypeTypedef = gql`
     entityType: EntityType!
   }
 
-  type PersistedEntityType implements PersistedEntityTypeInterface {
-    # INTERFACE FIELDS BEGIN #
-    """
-    The specific versioned URI of the entity type
-    """
-    entityTypeId: String!
-    """
-    The id of the account that owns this entity type.
-    """
-    ownedById: ID!
-    """
-    Alias of ownedById - the id of the account that owns this entity type.
-    """
-    accountId: ID!
-      @deprecated(reason: "accountId is deprecated. Use ownedById instead.")
-    """
-    The entity type
-    """
-    entityType: EntityType!
-    # INTERFACE FIELDS END #
-  }
-
-  type EntityTypeRootedSubgraph implements PersistedEntityTypeInterface {
-    """
-    Data types indirectly referenced by this entity type
-    """
-    referencedDataTypes(depth: Int): [PersistedDataType!]!
-    """
-    Property types directly or indirectly referenced by this entity type
-    """
-    referencedPropertyTypes(depth: Int): [PersistedPropertyType!]!
-    """
-    Link types directly or indirectly referenced by this entity type
-    """
-    referencedLinkTypes(depth: Int): [PersistedLinkType!]!
-    """
-    Entity types directly or indirectly referenced by this entity type
-    """
-    referencedEntityTypes(depth: Int): [PersistedEntityType!]!
-
-    # INTERFACE FIELDS BEGIN #
-    """
-    The specific versioned URI of the entity type
-    """
-    entityTypeId: String!
-    """
-    The id of the account that owns this entity type.
-    """
-    ownedById: ID!
-    """
-    Alias of ownedById - the id of the account that owns this entity type.
-    """
-    accountId: ID!
-      @deprecated(reason: "accountId is deprecated. Use ownedById instead.")
-    """
-    The entity type
-    """
-    entityType: EntityType!
-    # INTERFACE FIELDS END #
-  }
-
   extend type Query {
     """
     Get all entity types at their latest version.
     """
-    getAllLatestEntityTypes: [EntityTypeRootedSubgraph!]!
+    getAllLatestEntityTypes(
+      dataTypeResolveDepth: Int!
+      propertyTypeResolveDepth: Int!
+      linkTypeResolveDepth: Int!
+      entityTypeResolveDepth: Int!
+    ): Subgraph!
 
     """
     Get a entity type by its versioned URI.
```

Comment:
Should we update these docstrings accordingly in the typedefs?

Reply:
Appreciate you resolved this but I actually think it was a good idea to do right now. Have done so.